impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -27,6 +27,7 @@ namespace impeller {
 
 static std::set<std::string> kRequiredDeviceExtensions = {
     VK_KHR_SWAPCHAIN_EXTENSION_NAME,
+    VK_KHR_MAINTENANCE1_EXTENSION_NAME,  // For negative viewports.
```

Comment:
This shouldn't be necessary if we are specifying 1.1 API requirements.

Reply:
without this MotenVK complains :(, i think moten VK targets 1.0 by default.

Reply:
We have to opt-into 1.1 https://github.com/flutter/engine/blob/801935148c1d5a508550b63dfe1a8b18889b5c20/impeller/renderer/backend/vulkan/context_vk.cc#L275

Reply:
Thanks!